modules/rosa-sdpolicy-am-aws-compute-types-graviton.adoc

```diff
@@ -0,0 +1,180 @@
+
+// Module included in the following assemblies:
+//
+// * rosa_architecture/rosa_policy_service_definition/rosa-sdpolicy-am-compute-hcp.adoc
+:_mod-docs-content-type: CONCEPT
+[id="rosa-sdpolicy-aws-instance-types-graviton_{context}"]
+= AWS Graviton (ARM) instance types
+
+In addition to x86 architecture, {hcp-title} offers the following Graviton (ARM) worker node instance types and sizes:
+
+[NOTE]
+====
+Graviton instance types are only available for new clusters created after July 20, 2024. 
+====
+
```

Comment:
Remove extra line space

Reply:
from[ RHSG](https://redhat-documentation.github.io/supplementary-style-guide/#date-formats) should we use: `20 July, 2024`

Reply:
fixed on both